easybuild/easyconfigs/l/libsc/libsc-2.8-gmpolf-2017.10.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'libsc'
+version = '2.8'
+
+homepage = 'https://github.com/cburstedde/libsc'
+description = """ The SC Library provides support for parallel scientific
+applications."""
+
+toolchain = {'name': 'gmpolf', 'version': '2017.10'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/cburstedde/libsc/archive']
+sources = ['v%(version)s.tar.gz']
+patches = ['m4_files.patch']
```

Comment:
@rubendibattista Do you mind renaming this patch file to make it clear for which software & version this is?

Something like `libsc-2.8_m4-files.patch` makes sense.

Reply:
That patch is needed basically for all the versions. I submitted a PR to the original project to fix that upstream. Basically 99% of the build of `p4est` are made using an in-place `libsc` cloned as submodule, so it works despite not installing the missing m4 files. But in this case we're installing the two packages separately, so these macro files need to be installed correctly in order for `p4est` to find them correctly

Reply:
I indeed changed the name to `libsc-2.x_m4-files.patch`